README.rst

```diff
@@ -9,38 +9,26 @@ set of package requirements (for example, `antenna-selection/requirements.txt
 Setup Instructions
 ------------------
 As each demo may have a different set of package requirements, you may want to
-set up a virtual environment to contain said packages and run your demo within
-that environment.
+set up a `virtual environment <https://docs.ocean.dwavesys.com/en/latest/overview/install.html#python-virtual-environment>`_
+to contain said packages and run your demo within that environment.
 
-Virtual environment setup and package installation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Package Installation
+~~~~~~~~~~~~~~~~~~~~
 ::
 
-  git clone git@github.com:dwavesystems/demos.git
+  git clone https://github.com/dwavesystems/demos.git
   cd demos/<desired-demo>
-  python3 -m venv <desired-demo>_env
-  source <desired-demo>_env/bin/activate
   pip install -r requirements.txt
 
-Once the commands above have created and activated your virtual environment,
-and have installed the package requirements, you can run your demo.
+Please note that as the demos will be using the D-Wave Ocean SDK, they will
+likely need access to a D-Wave system. If you are accessing this for the first
```

Comment:
"accessing this" --> unclear what "this" refers to

How about, "Please note that to run demos that access a D-Wave system, you must:"?